src/layouts/MainLayout.js

```diff
@@ -41,9 +42,21 @@ const MainLayout = ({ children, pageContext, sidebarOpen = true }) => {
     }
   };
 
+  /*
+   * [VSU] some docs pages are being designed as JS for faster experimenting
+   * and will never have the frontmatter property
+   * Using regex for check to account for paths with and without trailing slash
+   */
+  const docsAsJS = ['/introduction-apm'];
+  docsAsJS.forEach((docUrl) => {
+    if (location.pathname.match(docUrl) && !hideNavs) {
+      hideNavs = true;
+    }
+  });
+
   useEffect(() => {
     setIsMobileNavOpen(false);
-    setSidebar(sidebarOpen);
+    setSidebar(hideNavs ? !hideNavs : sidebarOpen);
```

Comment:
nit: these are technically equivalent but i think it'd be more clear to just use `false`

```suggestion
    setSidebar(hideNavs ? false : sidebarOpen);
```

Reply:
Classic overthinking it.